cypress/cypress/integration/record.spec.ts

```diff
@@ -2261,6 +2261,61 @@ describe("Record", () => {
     cy.get("[data-cy=trim-video]").should("not.exist");
   });
 
+  it("guide silhouette should be visible while trimming a video", () => {
+    cySetup(cy);
+    cyMockUpload(cy);
```

Comment:
I would argue having a distinct function `cyMockUpload` separate from `cyMockDefault` and then 33 instances of the following:

```typescript
cyMockUpload(cy);
cyMockDefault(cy);
```

...is a violation of the DRY principal. Any reason not to make mocking the `upload` endpoint a part of the default mock and then getting rid of the distinct `cyMockUpload` (or at least don't export it)?

Probably we should do the same with `cyMockCancelUpload`

Reply:
33259fb